### PR TITLE
New working clones

### PR DESCRIPTION
**Pull request by clawgrip**
------------------
Galaxian (Electromar Spanish bootleg) [Juan Romero, ClawGrip]

### CODE REVIEW

File: src/mame/drivers/galaxian.cpp [hunk 1]
```diff
@@ -14741,6 +14757,7 @@ GAME( 1979, galaxianbl,  galaxian, galaxian,   galaxianbl, galaxian_state, init_
 GAME( 1979, galaxbsf2,   galaxian, galaxian,   galaxian,   galaxian_state, init_galaxian,   ROT90,  "bootleg",                                    "Galaxian (bootleg, set 3)",                                                                    MACHINE_SUPPORTS_SAVE )
 GAME( 1979, galaxianbl2, galaxian, galaxian,   galaxianbl, galaxian_state, init_galaxian,   ROT90,  "bootleg",                                    "Galaxian (bootleg, set 4)",                                                                    MACHINE_SUPPORTS_SAVE )
 GAME( 1979, galaxianbl3, galaxian, galaxian,   zerotime,   galaxian_state, init_galaxian,   ROT90,  "bootleg",                                    "Galaxian (Spanish bootleg)",                                                                   MACHINE_WRONG_COLORS | MACHINE_SUPPORTS_SAVE ) // unique color PROM that gives strange colors. Unfortunately PCB doesn't work so it cannot be verified
+GAME( 1980, galaxiem,    galaxian, galaxian,   galaxrf,    galaxian_state, init_galaxian,   ROT90,  "bootleg (Electromar)",                       "Galaxian (Electromar Spanish bootleg)",                                                        MACHINE_SUPPORTS_SAVE )
```
Comment (cuavas) [inline src/mame/drivers/galaxian.cpp:14760]: Make new clone set names by appending characters to the parent set name.

**Review by cuavas — CHANGES_REQUESTED** [review]
There are plenty of letters before the 16-character limit.  Just append letters to `galaxian` for the clone set name.

File: src/mame/drivers/galaxian.cpp (hunk 1 above)
Comment (clawgrip) [reply src/mame/drivers/galaxian.cpp]: Done, renamed from 'galaxiem' to 'galaxianem'.

**Comment by clawgrip** [comment]
> 
> 
> There are plenty of letters before the 16-character limit. Just append letters to `galaxian` for the clone set name.

Done, renamed from 'galaxiem' to 'galaxianem'. Thanks!

**Pull request merged**